notebooks/intercomparison-covermaps.ipynb

```diff
@@ -0,0 +1,1346 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    def rasterExtraction(image, resolution, f_collection):
</pre><p>nit: python function names usually use <code>lower_case_underscore</code> https://peps.python.org/pep-0008/#function-and-variable-names </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/nasaharvest/crop-mask/pull/228/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>